index.html

```diff
@@ -15,5 +16,81 @@
 
 <body>
 <!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
+
+<header class="head-nav">
```

Comment:
Is there any reason to add a specific class to your header if you only have one? You can just access it with the header selector.